test-cases/performance/perf-regression-nosqlbench-latency-mixed-50r-50w-aws.yaml

```diff
@@ -0,0 +1,17 @@
+test_duration: 360
```

Comment:
This is not how I meant  to split into configuration.
Please take a look in SCT where we have a folder names "Configurations".
There we add yamls that only adding parameters on top of existing yamls, then the pipelines are calling the test with 2 test yamls, the base one + the configuration one.

Then, you don't need to change one parameter in several places.

Reply:
Fixed (3 configuration files created for the 2 test cases and updated the pipeline definitions to include the config files)